### DISCUSSION TITLE
常用链接 — 博讯

### DISCUSSION THREAD

**Discussion by giscus** [hanbinjxnc/hanbinjxnc.github.io#2] [Announcements]
# 常用链接 — 博讯

Hello World！常用链接

https://blog.shibowl.top/wiki/%E5%B8%B8%E7%94%A8%E9%93%BE%E6%8E%A5/

**Comment by hanbinjxnc** [comment]
good，很好！

**Reply by hanbinjxnc** [reply]
非常好。
